### ISSUE TITLE
Suggestion to Add a Detail to INSTALL.md

### ISSUE THREAD

**Issue by AndrewLukan** [pret/pokefirered#678]
My apologies if this isn't proper procedure for making changes to a Github project. 

I went through the guide and encountered an issue at cloning and installing **agbcc**.

At step 2 of Installation, these four commands are instructed to be used:

**git clone https://github.com/pret/agbcc
cd agbcc
./build.sh
./install.sh ../pokefirered**

![image](https://github.com/user-attachments/assets/f4d04412-20b4-44c8-afa1-6510454d60d1)

As the guide has the copy icon, I pressed that and pasted all four into my Ubuntu terminal. For me, this was causing an error. Thankfully, GriffonR in the Discord told me to execute each one individually, and it was successful. I was able to build **pokefirered.gba** successfully thanks to them.

TL;DR, may I suggest a small text addition to that second step to execute each command one-by-one?

**Comment by Xeckt** [comment]
I don't see why, all of the commands are on new lines which tells you each one executes individually. If comands were to be executed together it would look something like:

`git clone https://<url> && cd agbcc; ./build.sh && ./install.sh ../pokefirered`

But it doesn't, and to be honest if you're building it yourself and regularly working it you won't be copy pasting these commands every time. It's negligible.

Copy pasting also does work, sometimes the pasting can be funky in the terminal so your issue was somewhere else, not with this.
